tools/workspace-plugin/src/generators/react-library/schema.json

```diff
@@ -18,17 +18,16 @@
     "owner": {
       "type": "string",
       "description": "Team owning the library. Will be written in CODEOWNERS",
+      "enum": [
```

Comment:
switched to enums in order to properly generate `Select` within `yarn generate`. This also enforces to use only enumerated strings instead of random string